src/components/maps/StaticControl/StaticControl.js

```diff
@@ -1,33 +1,29 @@
-import ReactDom from 'react-dom';
-
-import { MapControl, withLeaflet } from 'react-leaflet';
+import React, { useEffect } from 'react';
+import { createRoot } from 'react-dom/client';
+import { useMap } from 'react-leaflet';
 import L from 'leaflet';
-
 import './StaticControl.css';
 
+const StaticControl = ({ position, style, children }) => {
+  const map = useMap();
```

Comment:
Excellent! Just a day or two ago I realized that `useMap` might get around the problems with `useLeafletContext`.